101-automation/azuredeploy.json

```diff
@@ -0,0 +1,207 @@
+{
+"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+"contentVersion": "1.0.0.0",
+"parameters": {
+    "workspaceName": {
+        "type": "string",
+        "metadata": {
+            "description": "Workspace name"
+        }
+    },
+    "sku": {
+        "type": "string",
+        "allowedValues": [
+            "pergb2018",
+            "Free",
+            "Standalone",
+            "PerNode",
+            "Standard",
+            "Premium"
+        ],
+        "defaultValue": "pergb2018",
+        "metadata": {
+            "description": "Pricing tier: perGB2018 or legacy tiers (Free, Standalone, PerNode, Standard or Premium), which are not available to all customers."
+        }
+    },
+    "dataRetention": {
+        "type": "int",
+        "defaultValue": 30,
+        "minValue": 7,
+        "maxValue": 730,
+        "metadata": {
+            "description": "Number of days to retain data."
+        }
+    },
+    "location": {
```

Comment:
add defaultValue of resourceGroup().location

Reply:
Done